pkg/apis/policy/validation/validation_test.go

```diff
@@ -906,3 +906,30 @@ func TestValidatePSPRunAsGroup(t *testing.T) {
 		})
 	}
 }
+
+func TestValidatePSPSELinux(t *testing.T) {
```

Comment:
Thanks for adding the test, but it seems to be completely separate from the rest of this PR?

Reply:
Thanks for the review. Yes, I have added the test cases for different categories under the same PR. Can i separate them in 3 PR's?
Actually, I thought making 3 PR's just for the 3 test is seems more unfeasible.

Reply:
the validation tests can be in a PR together. the e2e test should be separate.

the e2e test requires the RunAsGroup feature to be on, so it should be a distinct test tagged with `[Feature:RunAsGroup]` until https://github.com/kubernetes/kubernetes/pull/73007 merges

Reply:
Done, Sorry for the late as I was busy last week.